### DISCUSSION TITLE
Expand record on authWithPassword does not work

### DISCUSSION THREAD

**Discussion by stefano-mek** [pocketbase/pocketbase#5550] [Q&A] [answered]
I'm trying to expand on a user record after logged, but doesn't expand.

This is my code:
```ts
this.pb.collection('users').authWithPassword(username, password, { expand: 'company' }))
```

Result: user record but no expand field in the result object

'company' is the name of the relation field on the users collection
'suppliers' is the table related to it

Thanks for the help

**Comment by ganigeorgiev** [comment, marked as answer]
There are probably at least dozens similar posts.

Make sure that your client request satisfies the View API rule of the expanded collection.

For more details please refer to https://pocketbase.io/docs/working-with-relations/#expanding-relations.

If that doesn't help, please export your collections configurations as json (Admin UI > Settings > Export collections).
